aten/src/TH/generic/THTensorMath.c

```diff
@@ -450,7 +450,7 @@ void THTensor_(take)(THTensor *r_, THTensor *src, THLongTensor *index)
   // Exceptions must not be thrown across OpenMP parallel sections, so we
   // record the value of the invalid index and throw the exception after the
   // loop.
-  int64_t invalidIdx = -1;
+  int64_t invalidIdxDim = -1;
```

Comment:
The naming is confusing because this isn't a dim (axis). Maybe invalidIdxOffset or invalidIdxPos